code/mob/new_player.dm

```diff
@@ -41,6 +41,11 @@ mob/new_player
 			mind = new(src)
 			keyd = mind.key
 
+		if (src?.client?.player) //playtime logging stuff
```

Comment:
```suggestion
		if (src.client?.player) //playtime logging stuff
```
`src` is never null except when you manually set it to null which hopefulyl doesn't happen here.